system_setup/server/server.py

```diff
@@ -88,3 +92,13 @@ def make_model(self):
     async def wait_for_cloudinit(self):
         self.cloud_init_ok = True
         return
+
+    async def start_site(self, runner: web.AppRunner):
+        port = self.opts.tcp_port
+        if port is None:
+            return await super().start_site(runner)
+
+        # Subiquity runs with root privileges. We don't wont outsiders to
```

Comment:
s/wont/want/

Reply:
+ I would clarify this is why we only listen on localhost.

Reply:
Thanks for noticing that typo.